lib/miq_automation_engine/engine/miq_ae_service.rb

```diff
@@ -342,6 +376,11 @@ def visible_domain?(dom)
       $log.warn "domain=#{dom} : is not viewable"
       false
     end
+
+    def locks_acquired(lock_name)
+      num_locks_acquired = MiqConcurrency::PGMutex.count_advisory_lock(lock_name)
```

Comment:
@dtrieu80 
This could be a 1 line function
MiqConcurrency::PGMutex.count_advisory_lock(lock_name)